### DISCUSSION TITLE
全聯父子的新野心｜對決Line Pay和街口！林敏雄年虧5億拚電支之王

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#709] [General]
我認為pxpay的成功本來就很令人驚訝，畢竟政府也很努力在推台灣pay，但是效果卻不盡人意，而且pxpay的客群很特別，是相對富裕但不習慣使用電子支付的中年人，不但忠誠度高而且數量龐大。全聯近幾年也很努力的擴大通路併購了大潤發、頂好等連鎖量販店，期待它之後的推出的更多服務。
網址: https://www.businessweekly.com.tw/focus/indep/6008386

**Comment by Yunhans** [comment]
我覺得現在xxpay有夠多能不能整合一下

**Comment by williamyen0623** [comment]
現在去全聯都還在用台灣pay，我也好想用用看pxpay

**Comment by manipulateed** [comment]
全聯的擴大跟選對領導人有很大的關係，讓這個品牌更加深名遠播ˋ和讓人喜愛。

**Comment by CPCpyt** [comment]
這是很大的突破，我想支付工具的加入只是開始，
或許能為全聯入主金融埋下成功地伏筆。
